docs/operate/get-started/supported-hardware/_index.md

```diff
@@ -25,99 +25,55 @@ prev: "/operate/get-started/setup/"
 next: "/operate/get-started/other-hardware/"
 ---
 
-Viam supports a wide variety of sensors, cameras, and other physical hardware, with APIs for each of the following types of hardware:
+## About the Viam Registry of supported hardware and software
 
-{{< cards >}}
-{{% relatedcard link="/dev/reference/apis/components/arm/" %}}
-{{% relatedcard link="/dev/reference/apis/components/base/" %}}
-{{% relatedcard link="/dev/reference/apis/components/board/" %}}
-{{% relatedcard link="/dev/reference/apis/components/button/" %}}
-{{% relatedcard link="/dev/reference/apis/components/camera/" %}}
-{{% relatedcard link="/dev/reference/apis/components/encoder/" %}}
-{{% relatedcard link="/dev/reference/apis/components/gantry/" %}}
-{{% relatedcard link="/dev/reference/apis/components/gripper/" %}}
-{{% relatedcard link="/dev/reference/apis/components/input-controller/" %}}
-{{% relatedcard link="/dev/reference/apis/components/motor/" %}}
-{{% relatedcard link="/dev/reference/apis/components/movement-sensor/" %}}
-{{% relatedcard link="/dev/reference/apis/components/power-sensor/" %}}
-{{% relatedcard link="/dev/reference/apis/components/sensor/" %}}
-{{% relatedcard link="/dev/reference/apis/components/servo/" %}}
-{{% relatedcard link="/dev/reference/apis/components/switch/" %}}
-{{% relatedcard link="/dev/reference/apis/components/generic/" %}}
-{{< /cards >}}
+Viam has a registry of supported hardware {{< glossary_tooltip term_id="module" text="modules" >}} that implement [standardized APIs](/dev/reference/apis/#component-apis) for each category of hardware {{< glossary_tooltip term_id="component" text="component" >}} (for example, the camera API).
+Any hardware that is not already supported by a Viam module can be added into Viam’s system of modular resources by [creating a new module](/operate/get-started/other-hardware/) that provides a driver for the hardware.
 
-These standardized APIs are implemented by {{< glossary_tooltip term_id="module" text="modules" >}} that provide drivers for specific models of hardware.
+The Viam Registry is the storage and distribution system for not just hardware modules but also software modules (called services), ML models, and ML model training scripts.
+You can browse the [Viam Registry in the Viam app](https://app.viam.com/registry?type=Module).
 
-Any hardware that is not already supported by a Viam module can be added into Viam's system of modular resources by [creating a new module](../other-hardware/) that provides a driver for the hardware.
+Some components are supported by drivers built into `viam-server`, so you won't find them in the registry page.
+You can browse all built-in and modular registry components on your machine's configuration page in the Viam app.
 
-{{% alert title="Tip" color="tip" %}}
+## Configure hardware on your machine
```

Comment:
didn't you say that you were inspired by github and some other site (doc detective?) that you wanted to have a more 'example' module that you actually installed on this page?  it's possible I'm inventing that but I really thought you and esha both mentioned that.

Reply:
So our original proposal (that had a more reference-style supported hardware page) included a quickstart that would guide a user through making a machine and adding a non-hardware component (modular or built-in, no specific preference there) to it. There wasn't a plan to put anything like that on this page specifically. I think something that specific would need to be in a separate guide and it'd get in the way of the generalizable instructions on this page.

Reply:
ok yeah I can see how there's a difference between the quickstart and the generalizable instructions.  can resolve.

Reply:
Okay. Also, Esha discussed the Test convo offline and decided to put three small screenshots in place of the GIF, and of course whether it's images or GIF they'll all show the addition of one example component so that should at least slightly cover this